test/Microsoft.ML.TestFramework/TestCommandBase.cs

```diff
@@ -1232,7 +1232,7 @@ public void CommandTrainMlrWithLabelNames()
             Done();
         }
 
-        [Fact]
+        [NotAppleSiliconFactAttribute("Temporarily skipping while helix issues are resolved. Tracked in issue #5845")]
```

Comment:
Why is this test still skipped here?

I think we should strive for not having the `NotAppleSiliconFactAttribute` class, if we can.

> Tracked in issue #5845

Should use full URLs to issues.

Reply:
Alright, it is no longer skipped. Added in the correct new baseline files for it.